testhelper/testhelper.go

```diff
@@ -48,19 +47,37 @@ func ResetFolder(path string) error {
 	return os.Mkdir(path, dirInfo.Mode())
 }
 
+// TODO 05/12/2022: Both type and constants will be deprecated when data ingestion via GRPC is available
+// only being used now for linter issues.
+type (
```

Comment:
@kkufieta This is copy & pasted from https://github.com/viamrobotics/rdk/pull/2271/files#diff-8de973b88008f6c9ebcfadf5c08cc3eec10c2ea4243f6a308830616752591d7aL1